namespaces/init.go

```diff
@@ -95,7 +103,11 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 
 	label.Init()
 
-	if err := mount.InitializeMountNamespace(rootfs,
+	if (cloneFlags & syscall.CLONE_NEWNS) == 0 {
```

Comment:
I'm okay with style, but it is not required here.
```
if (cloneFlags & syscall.CLONE_NEWNS) == 0 && container.MountConfig != nil {
    return fmt.Errorf("mount_config is set without mount namespace")
}
if err := mount.InitializeMountNamespace(rootfs,...
```
should be same.
Feel free to left it as-is, just FYI :)